st2debug/tests/integration/test_submit_debug_info.py

```diff
@@ -120,22 +120,55 @@ def test_create_archive_include_all(self):
 
     def _get_yaml_config(self):
         return {
-            'log_file_paths': [
+            'log_files_paths': [
```

Comment:
Can we please change this back to `log_file_path` so it's consistent with existing `config_file_paths` (or vice versa, make sure it's `files` in both places :))